src/renderer/tvgLoadModule.h

```diff
@@ -150,7 +150,7 @@ struct FontMetrics
 
     ~FontMetrics()
     {
-        free(engine);
+        tvg::free(engine);
```

Comment:
I think this should be applied with a separate PR. Please raise a PR if you want. Thanks.